octoprint_discordremote/command.py

```diff
@@ -476,14 +478,139 @@ def unzip(self, params):
                         with zip.open(f) as source:
                             file.write(source.read())
 
-                self.plugin.get_file_manager().remove_file('local', unzippable.rpartition('/')[2])
+                self.plugin.get_file_manager().remove_file('local', unzippable)
 
         except:
-            return None, error_embed(author=self.plugin.get_printer_name(), title="Bad zip file.",
-                                 description='In case of multi-volume files, one could be missing.')
+            self.plugin.get_file_manager().remove_file('local', unzippable)
+            return None, error_embed(author=self.plugin.get_printer_name(),
+                                     title="Bad zip file.",
+                                     description='In case of multi-volume files, one could be missing.')
+
+        filelist_string = ''
+        for f in filestounpack:
+            filelist_string += (f + '\n')
+
+        return None, success_embed(author=self.plugin.get_printer_name(), title='File(s) unzipped. ', description=filelist_string)
+
+    #check if received file is eligible for unzipping
+    def judge_is_unzippable(self, filename):
+
+        autounzip = self.plugin.get_settings().get(['auto_unzip'])
+        truncated = filename[:-4]
+
+        #file is a single zip
+        if filename.endswith('.zip'):
+            #unzip
+            if autounzip:
+                return True, success_embed(author=self.plugin.get_printer_name(),
+                                           title='File Received, unzipping...',
+                                           description=filename)
+
+            #inform the user that he can unzip the file himself
```

Comment:
Very much a nitpick, but I prefer to go with "they" instead of "he" or "she", and "themselves"

Reply:
Sure :)